jck/jck.mk

```diff
@@ -72,11 +72,9 @@ ifeq (,$(findstring testsuite, $(JCK_CUSTOM_TARGET)))
    override JCK_CUSTOM_TARGET := $(JCK_CUSTOM_TARGET) testsuite=RUNTIME
 endif
 
-ifndef JRE_IMAGE
-	JRE_ROOT := $(TEST_JDK_HOME)
-	JRE_IMAGE := $(subst j2sdk-image,j2re-image,$(JRE_ROOT))
-  JRE_COMMAND := $(Q)$(JRE_IMAGE)$(D)bin$(D)java$(Q)
-endif
+define JRE_COMMAND
+$(Q)$(JRE_IMAGE)$(D)bin$(D)java$(Q)
+endef
```

Comment:
I think we do not need this if block anymore.

Reply:
ah, cool, yes, see it in https://github.com/adoptium/TKG/pull/232 (somehow I thought it was just JRE_IMAGE defined in TKG)